.kokoro/samples-test.sh

```diff
@@ -21,6 +21,7 @@ export NPM_CONFIG_PREFIX=${HOME}/.npm-global
 # Setup service account credentials.
 export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/service-account.json
 export GCLOUD_PROJECT=long-door-651
+export GOOGLE_CLOUD_PROJECT_NUMBER=1046198160504
```

Comment:
This will be reset nightly by code generation, you could instead use a [pre-sample-test.sh](https://github.com/googleapis/nodejs-automl/blob/master/.kokoro/pre-samples-test.sh) file.

Reply:
Done